pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/PersistentTopics.java

```diff
@@ -662,6 +662,35 @@ public PersistentOfflineTopicStats getBacklog(@PathParam("property") String prop
         return internalGetBacklog(authoritative);
     }
 
+    @GET
+    @Path("/{property}/{cluster}/{namespace}/{topic}/ledger/{ledgerId}/entry/{entryId}/checkLedgerExists/{checkLedgerExists}/backlogSize")
```

Comment:
To me this is a very complicated api. Where is a user suppose to get ledger Id and entry Id? Can just a message Id work? what is the purpose of this `checkLedgerExists`?

Reply:
for  calculating backlog size across partitions,  Firstly, We had checked if the target ledgerId existed on topics across partitions, We  should not check If the ledger exists on partitioned topic lately, because target ledger is only exists on one partition.